bff_v0.py

```diff
@@ -0,0 +1,169 @@
+""" Quick'n'dirty mapping of bff for python before I can make it pure-rust
```

Comment:
This is fine, if you want to put it into a `scripts/` directory, but I feel like this is not really necessary. You could replace the entire thing with a few lines of bash:
```bash
aws cp "s3://bucket/input_dir/*.jsonl.gz" ./input_files
bff --bloom-filter-file filter.bff --bloom-filter-size <something> --expected-ngram-count <something> --output-directory ./output_files ./input_files/*.json.gz
rm -r input_files
aws cp ./output_files s3://bucket/output_dir
```